client/api/src/execution_extensions.rs

```diff
@@ -97,6 +99,7 @@ impl<Block: traits::Block> Default for ExecutionExtensions<Block> {
 		Self {
 			strategies: Default::default(),
 			keystore: None,
+			crypto_extension: RwLock::new(Arc::new(DefaultCryptoImpl)),
```

Comment:
I think it shouldn't be required since it's going to be rarely customized. We should either add it not as an extension but rather to `Externalities` trait (with default sane implementations) or we should allow overriding via `extensions_factory` - i.e. we register the default extension only if it's not present in the `Extensions` returned by `extensions_factory` invocation.